locales/fr/translation.json

```diff
@@ -3,7 +3,7 @@
   " I am 18 years old or older.": "J'ai plus de 18 ans",
   " I am between 13 and 17 years old. You'll need to have a parent or guardian agree to the terms below.": "J'ai entre 13 et 17 ans. Vous avez besoin d'avoir un parent ou un tuteur pour accepter les conditions ci-dessous.",
   " of ": "de",
-  " reports the data.": "Ramène les données",
+  " reports the data.": " Ramène les données",
```

Comment:
j'ai besoin de voir en contexte, je trouve pas sur l'app

Reply:
elle est pourrie cette phrase et j'ai enfin trouvé:

```
    var a = t("We can't be 100% certain of the data displayed here because of how ");
    var b = t(' reports the data.');
    ... 
    return a + sourceText + b;
```
Donc hors contexte pour nous car lié à l'uploader Tidepool. 
je fais au mieux